sebex/cmd/release.py

```diff
@@ -30,25 +31,70 @@ def status():
     else:
         success('There is no release pending at this moment, feel free to start one.')
 
+def gather_input() -> dict[Version, ProjectHandle]:
+    bumps = {}
+    log("hit enter when done", color='yellow')
+    while True:
+        value = click.prompt("Project", default="", show_default=False)
+        if value == "":
+            break
+        project = valid_project(value)
+        if project is None:
+            continue
+        # TODO at this moment all packages are bumped by one minor version
+        # TODO it would be better to specify different release versions for different packages
```

Comment:
I assume these things will be fleshed out in subsequent PRs?

Reply:
I should talk about this with @mat-hek but he's currently on vacation. From talking with @mickel8 I gathered that bumping everything by one minor version is practically the only use case at the moment so I wanted to commit what I have and do something else while I don't know for sure what's required

Reply:
👍🏻